cpp-lib/include/SNACCROSE.h

```diff
@@ -133,7 +133,7 @@ class ROSERejectChoice : public AsnType
    ROSERejectChoice() {Init();}
    ROSERejectChoice(const ROSERejectChoice& that);
 public:
-   virtual const char* typeName() const	{ return "ROSERejectChoice"; }
+   const char* typeName() const	{ return "ROSERejectChoice"; }
```

Comment:
missing "override" keyword

Reply:
If i add override to a single function, the compiler complains about not using override for the others.
So we will address override in BUILDSYS-310